prov/efa/Makefile.include

```diff
@@ -77,6 +77,7 @@ _efa_headers = \
 	prov/efa/src/rxr/rxr_pkt_cmd.h \
```

Comment:
`furthur` to `further` in commit message.

Reply:
fixed in new revision